### ISSUE TITLE
This package is being maintaned? 

### ISSUE THREAD

**Issue by joao-salomao** [DanSnow/vue-recaptcha#1000]
Looks like there no updates in a while, I'm wondering if will be safe to use it in a new project.

**Comment by DamianGlowala** [comment]
@DanSnow, I'm wondering whether you have any time scale for the v1.0.0 release and, most of all, time capacity to commit to maintaining this library?

**Comment by planigan** [comment]
An update on when the v1.0.0 release will be finished would be great.

**Comment by DanSnow** [comment]
Yes, this package is actively maintained. Closing this as the question has been addressed — recent releases and activity confirm ongoing maintenance.
